### ISSUE TITLE
Seed Settings

### ISSUE THREAD

**Issue by BlankSHC** [ryanxhr/POR#1]
Hi, how do you take the values of seed during training and seed+seed_offset during testing? When I take seed=[0,1,2,3,4], seed_offset=100, the result I get is very different from the result presented in the paper.
Thanks!

**Comment by marvin-oh** [comment]
I have the same issue with reproducting results.

**Comment by ryanxhr** [comment]
Hi, really sorry for the late reply

We apologize for the bug in our code from the previous version. We have updated the code.

Note that there is some mismatch between our provided code and the code that we used to produce the paper results. The copyright of our original code belongs to the previous company I worked for. Unfortunately, we didn't have permission to open-source it and the current github version is our re-implementation. Although we try to reimplement it as similar to the original version as possible, there is still one dataset, "hopper-medium-v2" datasest, where our re-implementation version achieves a 78.6 score, not able to reproduce 98.2 in the white paper. Other scores almost match the results in our paper (may have some minor difference due to random seeds)

Note that some configurations and experimental details are different from the original code (e.g., the value of $\alpha$, learning rate, and so on). We have updated our paper in the arxiv to reflect that and the mismatch results in the hopper-medium-v2 dataset (lower it from 98.2 to 78.6). If you want to compare POR with your work, you can refer to this paper version.

Let me know if you have any further questions!
